services/common/src/redux/utils/helpers.ts

```diff
@@ -622,22 +622,23 @@ export const getMineReportStatusDescription = (
   statusCode: MINE_REPORT_SUBMISSION_CODES,
   latestSubmission: IMineReportSubmission
 ) => {
-  const currentDate = moment().format("YYYY-MM-DD");
+  const update_timestamp = latestSubmission?.update_timestamp;
+  const updatedDate = moment(update_timestamp).format("YYYY-MM-DD");
```

Comment:
was making test fail. I _think_ update_timestamp is the value we should be using.